test/test_cloud_record.py

```diff
@@ -164,16 +165,16 @@ def setUp(self):
                          'PublicIPCount': None,
                          'Memory': 512,
                          'BenchmarkType': 'None',
-                         'Benchmark': 'None',
+                         'Benchmark': 0.0,
                          'ImageId': '\'scilin6\'',
                          'CloudType': 'OpenNebula'}
         self.cases = {}
         self.cases[self._msg1] = self._values1
         self.cases[self._msg2] = self._values2
         self.cases[self._msg3] = self._values3
 
-    def test_load_from_msg(self):
-        
+    def test_load_from_msg_value_check(self):
+        """Check for correct values in CloudRecords generated from messages."""
```

Comment:
I've changed this to be more about what we want to happen, rather than a fairly direct description of the code. Tests and their comments/docstrings can be quite useful when you need to check what _should_ be happening.